net-p2p/automatic/automatic-0.8.3.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit user eutils autotools
+
+DESCRIPTION="RSS downloader for Tranmission"
+HOMEPAGE="https://github.com/1100101/Automatic"
+SRC_URI="https://github.com/1100101/Automatic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+S="${WORKDIR}/Automatic-${PV}"
+
+RDEPEND="dev-libs/libxml2:2
+	dev-libs/libpcre:3
+	net-misc/curl"
```

Comment:
you sure these are not linked to? If they are needed at build time, don't forget the `DEPEND="${RDEPEND}"`